src/pages/Quiz/QuestionWrapper/index.js

```diff
@@ -63,27 +60,23 @@ const QuestionWrapper = (props) => {
 
   const classes = useStyles();
 
-  if (init) {
-    props.fetchTestDetails(testNumber);
-  }
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit